scripts/manage-db.py

```diff
@@ -84,13 +85,42 @@ def cleanup_releases_history(trans, dryrun=True):
 
     print "Total Deleted: %d" % total_deleted
 
+
+def extract_active_data(URL, loc="."):
+    """
+    Function was added in to enhance testing in Balrog's stage environment.
+
+    :param URL: Database. eg : mysql://balrogadmin:balrogadmin@balrogdb/balrog
+    :param loc: location where sqldump file must be created
+    :return: Stores sqldump data in dump.sql file in the specified location, if not specified stores in current directory
+            If dump.sql file already exists it will override that file and not append it.
+    """
+    data = re.split(':|/|@', URL)
```

Comment:
I know it's worked in your own testing, but there's a lot of edge cases when parsing URLs. Please use https://docs.python.org/2/library/urlparse.html instead.